tests/Paket.LoadingScripts.Tests/LoadingScriptTests.fs

```diff
@@ -0,0 +1,43 @@
+﻿module Paket.LoadingScriptTests
```

Comment:
Ok, good initiative, I think we need few integration tests with sample paket.dependencies and checking generated scripts are in place / matching expectation.